<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net&gt;</author>
 <comment>This file is part of the KDE project.
Copyright 2010 Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net&gt;

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, write to the Free Software
Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
</comment>
 <class>SortDetailsWidget</class>
 <widget class="QWidget" name="SortDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>60</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="m_useCustomLists">
     <property name="text">
      <string>Use custom list</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="m_customList">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="m_copyLayout">
     <property name="text">
      <string>Copy cell formatting (Borders, Colors, Text Style)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_useCustomLists</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_customList</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
